### PR TITLE
Community readiness improvements

### PR DESCRIPTION
**Pull request by rnovikovP**
- Contributing improvements
- Product clarification in the readme 
- remove old contributing to avoid confusion 
- standard code of conduct file added 

more changes will come later

### CODE REVIEW

**Review by denisok — DISMISSED** [review]

**Comment by denisok** [comment]
contributing in `.github` dir?  could we add Discord link somewhere as well ?

File: README.md [hunk 1]
```diff
@@ -5,7 +5,19 @@
 
 ![PMM](https://www.percona.com/sites/default/files/pmm-logo.png)
 
-See the [PMM 2.x docs](https://www.percona.com/doc/percona-monitoring-and-management/2.x/index.html) for more information.
+
+## Percona Monitoring and Management
+
+A single pane of glass to easily view and monitor the performance of your MySQL, MongoDB, PostgreSQL, and MariaDB databases.
```
Comment (PaulJacobs-percona) [inline README.md:11]:
```suggestion
A 'single pane of glass' to easily view and monitor the performance of your MySQL, MongoDB, PostgreSQL, and MariaDB databases.
```

Comment (PaulJacobs-percona) [reply README.md:11]: Because it's not literally a pane of glass, but a metaphor.

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -1,3 +1,42 @@
...
+
+If you find a bug in Percona Monitoring and Management  or one of the related projects, you should submit a report to that project's [JIRA](https://jira.percona.com) issue tracker.
+
+Your first step should be [to search](https://jira.percona.com/issues/?jql=project=PMM) the existing set of open tickets for a similar report. If you find that someone else has already reported your problem, then you can upvote that report to increase its visibility.
+
+If there is no existing report, submit a report following these steps:
+
+1. [Sign in to Percona JIRA.](https://jira.percona.com/login.jsp) You will need to create an account if you do not have one.
+2. [Go to the Create Issue screen and select the relevant project.](https://jira.percona.com/secure/CreateIssueDetails!init.jspa?pid=11600&issuetype=1&priority=3)
+3. Fill in the fields of Summary, Description, Steps To Reproduce, and Affects Version to the best you can. If the bug corresponds to a crash, attach the stack trace from the logs.
```
Comment (PaulJacobs-percona) [inline .github/CONTRIBUTING.md:28]:
```suggestion
3. Fill in the fields of *Summary*, *Description*, *Steps To Reproduce*, and *Affects Version* the best you can. If the bug corresponds to a crash, attach the stack trace from the logs.
```

Comment (PaulJacobs-percona) [reply .github/CONTRIBUTING.md:28]: Give a clue that these are fields in Jira.

**Review by PaulJacobs-percona — APPROVED** [review]

**Comment by denisok** [comment]
> > contributing in `.github` dir? could we add Discord link somewhere as well ?
> 
> @denisok this first iteration. Good catch about Discord. will add it to template to be added in all repos later

sure - that is why I have approved :)

**Review by denisok — APPROVED** [review]

**Review by BupycHuk — APPROVED** [review]

**Pull request merged**